src/ui/public/chrome/config/interval.html

```diff
@@ -1,7 +1,7 @@
 <kbn-timepicker
-  from="timefilter.time.from"
-  to="timefilter.time.to"
-  mode="timefilter.time.mode"
+  from="opts.timefilter.time.from"
```

Comment:
See above. Given that the time picker is global we don't need to attach opt.timefilter to every scope